packages/console/src/pages/Connectors/components/Guide/index.tsx

```diff
@@ -74,6 +74,13 @@ const Guide = ({ connector, onClose }: Props) => {
     });
   }, [formItems, reset, configTemplate, target, isSocialConnector, isStandard]);
 
+  const parseConfig = useCallback(
```

Comment:
this can be a hook?

Reply:
True, wrapped the connector form config parser as a hook.